docs/docs/platform/quickstarts/vue.mdx

```diff
@@ -0,0 +1,717 @@
+on port 3000. Open [http://localhost:3000](http://localhost:3000) from your
+browser to check this out.
+
+### Configure Nhost with Vue
+
+To work with Nhost from within our Vue app, we'll use the
+[Vue SDK](https://github.com/nhost/nhost/tree/main/packages/react) provided
+by Nhost. It's a wrapper around the
+[Nhost JavaScript SDK](https://github.com/nhost/nhost/tree/main/packages/nhost-js) which
+gives us a way to interact with our Nhost backend using Vue composables.
+
+You can install the Nhost Vue SDK with:
+
+<Tabs groupId="package-manager">
+  <TabItem value="npm" label="npm" default>
+
+```bash
+npm install @nhost/vue graphql
+```
+
+  </TabItem>
+  <TabItem value="yarn" label="Yarn">
+
+```bash
+yarn add @nhost/vue graphql
+```
+
+  </TabItem>
+  <TabItem value="pnpm" label="pnpm">
+
+```bash
+pnpm add @nhost/vue graphql
+```
+
+  </TabItem>
+</Tabs>
+
+Next, open your `src/main.ts` file as we'll now configure Nhost inside our app.
+
+The Nhost Vue SDK comes with a `NhostClient` that can be loaded into the Vue application as a plugin.
+It makes the authentication state and all the provided Vue composables available in our
+application.
+
+Use the following code to instantiate a new Nhost client and link it to your
+Nhost backend:
+
+```tsx title="src/main.ts"
+import { createApp } from 'vue'
+import { createRouter, createWebHistory } from 'vue-router'
+import routes from 'virtual:generated-pages'
+// highlight-start
+import { NhostClient } from '@nhost/vue'
+// highlight-end
+import App from './App.vue'
+
+import '@unocss/reset/tailwind.css'
+import './styles/main.css'
+import 'uno.css'
+
+// highlight-start
+const nhost = new NhostClient({
+  backendUrl: import.meta.env.VITE_NHOST_URL
+})
+// highlight-end
+
+const app = createApp(App)
+const router = createRouter({
+  history: createWebHistory(import.meta.env.BASE_URL),
+  routes
+})
+app
+  .use(router)
+  // highlight-start
+  .use(nhost)
+// highlight-end
+app.mount('#app')
+```
+
+Finally, make sure to create an environment variable named
+`VITE_NHOST_URL` to store your Nhost backend URL:
+
+```bash title=".env"
+VITE_NHOST_URL=YOUR_NHOST_BACKEND_URL
+```
+
+You can find your Nhost backend URL for your project from [your dashboard](https://app.nhost.io) as shown below:
+
+![Nhost Backend URL](/img/quickstarts/app-dashboard.png)
+
+## Build the app
+
+### Add authentication
+
+#### 1. Sign-up
+
+The next step is to allow our users to authenticate into our application.
+Let's start with implementing the sign-up process.
+
+For that, we'll use the `useSignUpEmailPassword` composable provided by the Nhost
+Vue SDK within a `/sign-up` page.
+
+Let's create a new page in your project using the following code:
+
+```markup title="src/pages/sign-up.vue"
+<script setup lang="ts">
+  import { ref } from 'vue'
+  import { useSignUpEmailPassword } from '@nhost/vue'
+  import { useRouter } from 'vue-router'
+  const inputClass =
```

Comment:
Tailwind is intentionally not used in React and Next.js quickstart guides to reduce the mental overhead. Could you please migrate to CSS modules?

Reply:
I then need to set and maintain the 'starter' Vue project - for the moment we are starting from [a pre-existing boilerplate](https://github.com/antfu/vitesse-lite)